integration-test/1491-generic-network-fixes.py

```diff
@@ -0,0 +1,4231 @@
+                'lanes:psv:forward': u'1',
+                'lit': u'yes',
+                'maxspeed': u'50',
+                'name': u'Rue du Mont-Blanc',
+                'oneway': u'no',
+                'ref': u'101',
+                'source': u'openstreetmap.org',
+                'surface': u'asphalt',
+                'trolley_wire': u'yes',
+            }),
+            dsl.relation(1, {
+                'name': u'Genève — Meyrin — (F)',
+                'network': u'ch:regional',
+                'ref': u'101',
+                'route': u'road',
+                'source': u'openstreetmap.org',
+                'type': u'route',
+            }, ways=[224035836]),
+        )
+
+        self.assert_has_feature(
+            z, x, y, 'roads', {
+                'id': 224035836,
+                'network': u'CH:regional',
+            })
+
+    def test_conational(self):
+        import dsl
+
+        z, x, y = (16, 19685, 31498)
+
+        self.generate_fixtures(
+            dsl.is_in('CO', z, x, y),
+            # https://www.openstreetmap.org/way/346899351
+            dsl.way(346899351, dsl.tile_diagonal(z, x, y), {
+                'alt_name': u'Saravena - Arauquita',
+                'highway': u'secondary',
+                'lanes': u'2',
+                'name': u'Diagonal 30',
+                'oneway': u'no',
+                'ref': u'66',
+                'source': u'openstreetmap.org',
+                'surface': u'asphalt',
+            }),
+            dsl.relation(1, {
+                'description': u'Tramo La Lejía-Saravena',
+                'name': u'66-04',
+                'network': u'co:national',
+                'operator': u'Invias',
+                'ref': u'66',
+                'route': u'road',
+                'source': u'openstreetmap.org',
+                'type': u'route',
+            }, ways=[346899351]),
+        )
+
+        self.assert_has_feature(
+            z, x, y, 'roads', {
+                'id': 346899351,
+                'network': u'CO:national',
+            })
+
+    def test_conationalold(self):
+        import dsl
+
+        z, x, y = (16, 19495, 30705)
+
+        self.generate_fixtures(
+            dsl.is_in('CO', z, x, y),
+            # https://www.openstreetmap.org/way/388323188
+            dsl.way(388323188, dsl.tile_diagonal(z, x, y), {
+                'highway': u'secondary',
+                'source': u'openstreetmap.org',
+            }),
+            dsl.relation(1, {
+                'name': u'49-03',
+                'network': u'co:national:old',
+                'operator': u'Concesión departamental',
+                'ref': u'49',
+                'route': u'road',
+                'source': u'openstreetmap.org',
+                'type': u'route',
+            }, ways=[388323188]),
+        )
+
+        self.assert_has_feature(
+            z, x, y, 'roads', {
+                'id': 388323188,
+                'network': u'CO:national:old',
```

Comment:
okay...

Reply:
I wasn't sure what to do with this one... Leave it as-is, knowing it won't match anything? Or remove it? Probably shouldn't normalise it to `CO:national`...

Reply:
I'd leave as-is.

It'll render as a generic shield which I think in this case is fine. This occasionally happens in the USA as well when the "old" route stays signed for a few years while people get used to the "new" route (which should receive the proper artwork).